util-bukkit/src/main/i18n/templates/strings.properties

```diff
@@ -0,0 +1,1641 @@
++ key naming convention, using example "firstKeyword.secondKeyword.thirdKeyword"
++ all names should follow the camelCase standard
++
++ firstKeyword should always be the group the string is in, example: "settings, ctw, classes, moderation"
++ secondKeyword should always be the name of the function the string belongs, example "moderation.unmute"
++ thirdKeyword is used for telling which part of the function the string belongs to, example "moderation.unmute.sender" or "moderation.unmute.target"
++ a thirdKeyword is only necessary if multiple strings belong to one function
++
++ long key names should always be avoided for easier reading/searching
++ optionally a explanation can be tagged to a string if needed
++
++ NOTE: Exceptions from these rules happen, death messages beeing one example.
+
++ EACH STRING WITH VARIABLES NEEDS TO HAVE A EXPLANATION FOR EACH VARIABLE IN THE FORM OF:
++ # {0} = explanation
++ # {1} = explanation
++ # {n}...
+
++ NOTE: Using "+" in front of comments will make them not appear on Crowdin or in any other place
+
+
++           -----ADMINISTRATION-----
++ Strings used in administrative tools that does NOT handle players
+
+admin.reload = Configuration reloaded
```

Comment:
We can just remove this one, use `command.admin.pgm` instead.

Reply:
Not sure i understand, remove the string? Or change the key

Reply:
It’s ok, just leave it